### PR TITLE
docs: update contribution guidelines for video challenges

### PR DESCRIPTION
**Pull request by beaucarnes**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

File: docs/how-to-help-with-video-challenges.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    - '6'
+    - '7'
+  solution: 3
+```
+
+For more examples, you can look at the markdown files for the following video course. All the challenges already have questions: [Python for Everybody Course](https://github.com/freeCodeCamp/freeCodeCamp/tree/next-python-projects/curriculum/challenges/english/07-scientific-computing-with-python/python-for-everybody)
+
+## Open a pull request
+
+After creating one or more questions, you can commit the changes to a new branch and open a pull request. Make sure that you target the branch 'next-python-projects' and NOT 'master.
```
Comment (ojeytonwilliams) [inline docs/how-to-help-with-video-challenges.md:143]:
```suggestion
After creating one or more questions, you can commit the changes to a new branch and open a pull request. Make sure that you target the branch 'next-python-projects' and NOT 'master'.
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
Looks good, I only found one missing '.

One though I had was: would it be useful to parse the questions and answers as Markdown?  I could see about adding that to the challenge parser if it would help.

**Comment by raisedadead** [comment]
> would it be useful to parse the questions and answers as Markdown? I could see about adding that to the challenge parser if it would help.

Yes, that would be nice. We will have video courses for most challenges (Python is just the starting point).

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by raisedadead — APPROVED** [review]

**Pull request merged**